pages/dao/[dao]/governance/settings/SettingsPage.module.scss

```diff
@@ -61,6 +61,10 @@
     flex-direction: row;
   }
 
+  > div {
+    margin-bottom: 24px;
+  }
+
   > div:not(:last-of-type) {
     margin-right: 24px;
   }
```

Comment:
I consider we can achieve same behaviour by providing class names to nested components.